app/webpacker/controllers/cookie-acceptance_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+
+    static targets = ["overlay"]
+
+    connect() {
+        this.checkforCookie();
+    }
+
+    checkforCookie() {
+        var cookie = document.cookie;
+        var cookieFrags = cookie.split(';');
+        for(var i=0;i<cookieFrags.length;i+=1) {
+            if(cookieFrags[i].indexOf('GiTBetaCookie=Accepted') > -1) {
+                return;
+            }
+        }
+        this.showDialog();
+    }
+
+    accept(event) {
+        event.preventDefault();
+        this.overlayTarget.style.display = "none";
+        document.cookie = "GiTBetaCookie=Accepted; expires=Fri, 31 Dec 2021 12:00:00 UTC";
```

Comment:
Why are we setting it to an absolute date?

Reply:
Discussed with Myles and Lizzie, at the moment it's as requested

Reply:
Ok, I guess it's getting removed after private beta anyway

Reply:
yeah, I considered like ... 3 months from todays date for example but ... I guess it doesn't matter so long as this all comes out when it goes public.